Core/Core/Planner/Model/UseCase/GetPlannables.swift

```diff
@@ -88,11 +89,42 @@ extension APICalendarEvent: PlannableItem {
     public var isHidden: Bool { hidden == true }
 }
 
+extension APIPlannerNote: PlannableItem {
+    public var plannableID: String { id }
+    public var plannableType: PlannableType { .planner_note }
+    public var htmlURL: URL? { nil }
```

Comment:
**TODO:** We need to validate using those values for those fields.

Reply:
AFAIK, `htmlURL` is used only for entities which have standalone pages on the web, too. (like Calendar Events have, but Calendar ToDos don't)
So it's expected that this would be `nil`. @vargaat could you please confirm this?

Reply:
Yes, I think that's correct. Despite this, when it comes to API entities an optional is always the safest method.